### ISSUE TITLE
[Bug]: error npm install @storybook/addon-mdx-gfm@7.6.18

### ISSUE THREAD

**Issue by kantesilvestrsTWL** [storybookjs/storybook#26929]
### Describe the bug

When installing `@storybook/addon-mdx-gfm@7.6.18` error is thrown `287 verbose stack Error: Unsupported URL Type "workspace:": workspace:*`

### To Reproduce

npm install @storybook/addon-mdx-gfm@7.6.18

### System

```bash
node v20.11.0
npm 10.2.4
```

### Additional context

```
$ npm install
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*
```

**Comment by bbkx226** [comment]
@marcomontalbano Thanks!

**Comment by miladpav** [comment]
It seems this issue related to some sort of changes in major version API's , in releases sequence `8.0.0-beta.4` is next release of `7.6.17`. so when you change `addon-mdx-gfm` to `8.0.0-beta.4` up to `8.0.9` versions works fine. 
We face this issue from 1 week to 3 week ago from this comment. 

I found 4 solutions to resolve this.
Solutions:

1. If you want work with exact version of your dependencies like `7.6.16`. You can first install single `addon-mdx-gfm` then install whole package
```bash
npm install @storybook/addon-mdx-gfm@7.6.18 --loglevel verbose && npm install --loglevel verbose
```
OR
```bash
yarn add @storybook/addon-mdx-gfm@7.6.18 --verbose && yarn install --verbose
```
2. Like what storybook website said `Once you've made the necessary changes, you can remove the addon from your package.json and storybook config.` you can just remove that.

3. as what @marcomontalbano said, ignore this addon and handle with `remark-gfm`. refrence of storybook website: https://storybook.js.org/docs/writing-docs/mdx#markdown-tables-arent-rendering-correctly

4. Like @cw-natejohnson said just upgrade to version `8.*.*`. `"@storybook/addon-mdx-gfm": "^8.0.9",`
 https://github.com/storybookjs/storybook/issues/26932#issuecomment-2077227180

**Comment by JReinhold** [comment]
This should be fixed now in version `7.6.19`. https://github.com/storybookjs/storybook/releases/tag/v7.6.19

Run `npx storybook@7.6.19 upgrade` to upgrade.
